docs-search/common/src/Docs/Search/Meta.purs

```diff
@@ -6,23 +6,21 @@ import Data.Codec.JSON as CJ
 import Data.Codec.JSON.Record as CJ.Record
 import Docs.Search.Config as Config
 import Docs.Search.Loader as Loader
-import Docs.Search.Types (PackageName)
-import Docs.Search.Types as Package
 import Effect.Aff (Aff, catchError)
 
 type Meta =
-  { localPackageName :: PackageName
+  {
```

Comment:
Is this going to be used at a later point? Or can it be removed?

Reply:
I would like to remove it but didn't want to expand the refactoring any further. I can have a go at it if you'd like

Reply:
Done